docs/testplan/dash/Dash-ACL-Tag-test-plan.md

```diff
@@ -0,0 +1,171 @@
+
+### Testbed
+The test will run on all DASH testbeds.
+
+### Setup configuration
+The tests are based on existing DASH ACL tests and extend their functionality.
+
+No setup pre-configuration is required, the tests will configure and clean up all the configuration.
+The configured IP addresses used in the test will be randomized.
+All test configurations will be done at the beginning and all test clean-ups will be after the end of test executions.
+
+Common tests configuration:
+- Create base vnet configurations
+- Create base ACL group, all tests ACL Tags, all tests ACL rules
+- Bind ACL in/out tables to ENI and ACL group
+
+Common tests cleanup:
+- Remove ACL binding
+- Remove ACL Tags, rules, group
+- Remove vnet configurations
+
+## Test
+
+## Test cases
+
+Each test case will be additionally tested by the loganalizer and core dump validation.
+
+### Test case \#1 - ACL Tag
+
+#### Test objective
+
+Verify functionality of ACL Tag configuration with 2 IPs.
+
+#### Test steps
+
+- setup
+    - Create src_addr Tag with 2 IP addresses.
+    - Create dst_addr Tag with 2 IP addresses.
+    - Create a Rule with these Tags.
+- validation
+    - Send 3 packets. Two of them should be received as matched by IP address.
+- teardown
+    - Remove Rule.
+    - Remove Tag.
+
+### Test case \#2 - ACL Multi Tag
+
+#### Test objective
+
+Verify functionality of ACL Tag, where Rule contains 2 Tags.
+
+#### Test steps
+
+- setup
+    - Create 2 src_addr Tags with a single IP address each.
+    - Create 2 dst_addr Tags with a single IP address each.
+    - Create a Rule with these Tags.
+- validation
+    - Send 2 packets, which should be received as matched by IP address.
+- teardown
+    - Remove Rule.
+    - Remove Tags.
+
+### Test case \#3 - ACL Tag Order
+
+#### Test objective
+
+Verify functionality of ACL Tag configuration where Rule created before Tag.
+
+#### Test steps
+
+- setup
+    - Create a Rule with a Tag.
+    - Create src_addr Tag with 2 IP addresses.
+- validation
+    - Send packet, which should be received as matched by IP address.
+- teardown
+    - Remove Tag.
+    - Remove Rule.
+
+### Test case \#4 - ACL Multi Tag Order
+
+#### Test objective
+
+Verify functionality of ACL Tag configuration where Rule created before Tags.
+
+#### Test steps
+
+- setup
+    - Create a Rule with 2 Tags.
+    - Create 2 src_addr Tags with a single IP address each.
+- validation
+    - Send 2 packets, which should be received as matched by IP address.
+- teardown
+    - Remove Tags.
+    - Remove Rule.
+
+### Test case \#5 - ACL Tag Update
+
+#### Test objective
+
+Verify functionality of ACL Tag configuration where Tag updated.
+
+#### Test steps
+
+- setup
+    - Create src_addr Tag with IP address.
+    - Create a Rule with this Tag.
+    - Update src_addr Tag with another IP address.
```

Comment:
Can you add a step for removing addr from a tag?

Reply:
Hi @prsunny , I have opened a new PR https://github.com/sonic-net/sonic-mgmt/pull/11094 for this test plan because Anton is not available now. And I have added the test case for removing address. Please review.